web/srv/server.go

```diff
@@ -44,6 +48,15 @@ type (
 
 // this is called by the HTTP server to actually respond to a request
 func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	if !s.reHost.MatchString(req.Host) {
+		error := fmt.Sprintf(`It appears that you are trying to reach this service with a host of '%s'.
+This does not match /^(localhost|127\.0\.0\.1|linkerd-web\.linkerd\.svc\.cluster\.local|\[::1\]):\d+$/ and has been denied for security reasons.
```

Comment:
Should this message include the actual regex instead of the default one?

Reply:
Yes, good catch :+1: